public/test/mock-api/scenarios/alerting/silences.ts

```diff
@@ -0,0 +1,5 @@
+import { silencesListHandler } from 'app/features/alerting/unified/mocks/server/handlers/silences';
+
+export const SILENCES_SCENARIOS = {
```

Comment:
This might be verbose but it keeps things separated nicely, similar to the handlers. Thoughts?